docs/database/tables/STAGE_YAHOO_HISTORICAL.sql

```diff
@@ -1,17 +1,18 @@
 CREATE TABLE STAGE_YAHOO_HISTORICAL(
-    HASH              NUMERIC(18),
-    SYMBOL            VARCHAR(20) NOT NULL,
-    SYM_ID            NUMERIC(18),
-    PER               VARCHAR(20) NOT NULL,
-    DATE              DATE NOT NULL,
-    TIME              TIME NOT NULL,
-    OPEN              NUMERIC(18),
-    HIGH              NUMERIC(18),
-    LOW               NUMERIC(18),
-    CLOSE             NUMERIC(18),
-    VOLUME            NUMERIC(18),
-    ACTIVE_FLAG       CHAR(1) DEFAULT 'Y' NOT NULL CHECK (ACTIVE_FLAG in ('Y','N') ),
-    ACTIVE_REASON     VARCHAR(20),
+    SYH_ID              NUMERIC(18) NOT NULL,
+    HASH                NUMERIC(18),
+    SYMBOL              VARCHAR(20) NOT NULL,
+    SYM_ID              NUMERIC(18),
+    PER                 VARCHAR(20) NOT NULL,
+    LOAD_DATE           TIMESTAMP WITH TIME ZONE NOT NULL DEFAULT NOW(),
+    OPEN                NUMERIC(18),
+    HIGH                NUMERIC(18),
+    LOW                 NUMERIC(18),
+    CLOSE               NUMERIC(18),
+    VOLUME              NUMERIC(18),
+    ACTIVE_FLAG         CHAR(1) DEFAULT 'Y' NOT NULL CHECK (ACTIVE_FLAG in ('Y','N') ),
+    ACTIVE_REASON       VARCHAR(20),
 
-    CONSTRAINT IPK_STAGE_YAHOO_HISTORICAL PRIMARY KEY (SYMBOL, PER, DATE, TIME, ACTIVE_FLAG)
-);
+    UNIQUE (SYMBOL, PER, CREATION_TM, ACTIVE_FLAG),
+    CONSTRAINT IPK_STAGE_YAHOO_HISTORICAL PRIMARY KEY (SYH_ID)
```

Comment:
Hibernate requires PK on a table. And using it instead of raw JDBC queries will reduce boilerplate.